src/Public/OpenApi.ps1

```diff
@@ -145,6 +160,18 @@ function Enable-PodeOpenApi {
         }
         $PodeContext.Server.OpenAPI.externalDocs = $PodeContext.Server.OpenAPI.hiddenComponents.externalDocs[$ExternalDoc]
     }
+    if ( $EnableSchemaValidation) {
+        #Test-Json has been introduced with version 6.1.0
+        if ($PSVersionTable.PSVersion -ge [version]'6.1.0') {
+            $PodeContext.Server.OpenAPI.hiddenComponents.schemaValidation = $EnableSchemaValidation.ToBool()
+        } elseif ($EnableSchemaValidation.ToBool()) {
```

Comment:
Could this not just be an simple `else` instead, as we're already in the scope of `if ($EnableSchemaValidation)`?

Reply:
elseif ($EnableSchemaValidation.ToBool())  cover -EnableSchema:$false

Reply:
`if ($EnableSchemaValidation)` on it's own won't be entered if either `-EnableSchemaValidation` isn't supplied, or if `-EnableSchemaValidation:$false` is set.

Reply:
I wish that was the case. In my experience return true if the switch is present.
But let me try with a sample

Reply:
I did throw together the below to double check, and only the top to write out:
```powershell
function test-me {
    param(
        [parameter()]
        [switch]
        $things
    )

    if ($things) {
        write-host "hello - $things"
    }
}

test-me -things
test-me -things:$true
test-me -things:$false
test-me
```

Reply:
yes you are right the issue is not with if but when you try to assign to a boolan variable

Reply:
Yeah, to be explicitly a bool for a variable then you would need to use either IsPresent or ToBool(), like you're using. In an `if` it just does this for you automagically.